poly-synth/index.html

```diff
@@ -20,23 +20,55 @@
     <script src="src/poly-synth-demo.js"></script>
     <script src="src/poly-synth.js"></script>
     <script src="src/poly-synth-voice.js"></script>
+    <style>
+      #container {
+        display: flex;
+        flex-direction: row;
+      }
+      .controls {
+        border-style: solid;
```

Comment:
I think the border makes it clearer which ADSR parameters refer to the filter and which refer to the amplifier